kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3OutputStream.java

```diff
@@ -280,6 +284,14 @@ public void abort() {
     }
   }
 
+  public OutputStream wrappedForCompression() {
+    if (this.compressionFilter == null) {
```

Comment:
nit: let's keep a uniform style in critical classes such as this one. `this.` is used only in constructors